src/components/Questions.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react'
+import Answers from './Answers'
+
+class Questions extends Component {
+  constructor() {
+    super()
+    this.loadQuestions = this.loadQuestions.bind(this)
+  }
+
+  loadQuestions() {
+    return this.props.questions.map(question => {
```

Comment:
This isn't really solving the problem. The point of a component is to re-use the code that applies to multiple elements on the page. A `Questions` component isn't re-using anything. It's just moving the code from one file to another. There should be a `Question` component that represents each question, since all of the code for each question will be exactly the same. We shouldn't have to map over an array to render a component. 

Again, in your quiz component, we want to render something like:

```javascript
questions.map((question, index) => {
    <Question title={question.title} key={question.id} />
    <div className="answer-group">
        (questions.answers).map((answer, index) => {
            <Answer title={answer.title} />
        })
    </div>
})
```

The Question component can be as simple as returning a paragraph tag that contains the title:

```javascript
return (<p>{this.props.title}</p>)
```

And the answer component can be as simple as:

```javascript
return (<p>{answer.title}</p>
```